.github/workflows/push-tests.yml

```diff
@@ -49,6 +49,8 @@ jobs:
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: us-west-2
+        role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+        role-duration-seconds: 900  
```

Comment:
For my own knowledge, what is the purpose of role-duration-seconds?

Reply:
It controls how long the session is valid for before requiring re-authentication. It's a security measure to prevent leaked sessions tokens from causing too much damage. There may be other reasons but this is the primary one I can think of. 

AWS may also have a limit to the number of active sessions allowed.